components/Footer/Footer.js

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import CorrectCounter from "../CorrectCounter/CorrectCounter";
+
+export default function Footer({
+  getIncorrectFlashcardsFromCollection,
+  getCorrectFlashcardsFromCollection,
+  id,
+  actionMode,
+  changeActionMode,
+}) {
+  const router = useRouter();
+
+  return (
+    <nav>
+      <StyledList>
+        <ListItem
+          onClick={() => changeActionMode("default")}
+          $active={router.pathname.endsWith("/to-learn")}
+        >
+          <CorrectCounter
+            isFooter={true}
+            variant="incorrect"
+            getIncorrectFlashcardsFromCollection={
+              getIncorrectFlashcardsFromCollection
+            }
+            id={id}
+            getCorrectFlashcardsFromCollection={
+              getCorrectFlashcardsFromCollection
+            }
+            actionMode={actionMode}
+          />
+        </ListItem>
+        <ListItem
+          onClick={() => changeActionMode("default")}
+          $active={router.pathname.endsWith("/[id]")}
+        >
+          <CorrectCounter
+            isFooter={true}
+            variant="all"
+            getIncorrectFlashcardsFromCollection={
+              getIncorrectFlashcardsFromCollection
+            }
+            id={id}
+            getCorrectFlashcardsFromCollection={
+              getCorrectFlashcardsFromCollection
+            }
+            actionMode={actionMode}
+          />
+        </ListItem>
+        <ListItem
+          onClick={() => changeActionMode("default")}
+          $active={router.pathname.endsWith("/learned")}
+        >
+          <CorrectCounter
+            isFooter={true}
+            variant="correct"
+            getIncorrectFlashcardsFromCollection={
+              getIncorrectFlashcardsFromCollection
+            }
+            id={id}
+            getCorrectFlashcardsFromCollection={
+              getCorrectFlashcardsFromCollection
+            }
+            actionMode={actionMode}
+          />
+        </ListItem>
+      </StyledList>
+    </nav>
+  );
+}
+
+const StyledList = styled.ul`
+  list-style: none;
+  display: flex;
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4rem;
+  z-index: 1;
+  background-color: #fff;
+`;
+
+const ListItem = styled.li`
+  width: 33.33%;
+  height: 100%;
+  text-align: center;
+  border: 1px solid #000;
+  border-left: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${(props) => (props.$active ? "#e6e6e6" : "#fff")};
```

Comment:
For consistency you could consider writing it like this:

```suggestion
  background-color: ${({ $active }) => ($active ? "#e6e6e6" : "#fff")};
```

Reply:
Oh yes! To be honest, that's something where I'm often unsure about the correct syntax and have to keep looking at it, but I could have written it often enough. ;-) I'll of course take your suggestion, thank you.